src/main/java/com/fnp/Map Interface/Main.java

```diff
@@ -0,0 +1,48 @@
+import employeedb.*;
+import java.util.*;
+import java.io.*;
+import java.lang.*;
+
+class Main{
+	
+	public static void main(String [] args){
+		
+		HashMap<Integer, Employee> map = new HashMap<>();
+				
+		Employee john= new Employee();
+		john.setId(2);
+		john.setName("John");
+		john.setDept("Tech");
+		john.setPhone(747386537);
+		john.setSal(35000);
+		
+		Employee jacob = new Employee();
+		jacob.setId(1);
+		jacob.setName("Jacob");
+		jacob.setDept("Ecom");
+		jacob.setPhone(73867);
+		jacob.setSal(35000);
+		
+		Employee james = new Employee();
+		james.setId(3);
+		james.setName("James");
+		james.setDept("Design");
+		james.setPhone(738643);
+		james.setSal(35000);
+		
+		map.put(1,john);
+		map.put(2,jacob);
+		map.put(3,james);
+		
+		for(Map.Entry<Integer,Employee> m:map.entrySet()){
```

Comment:
what other ways do we have other than iterating entry set..please check that also @saranshvijay ..for now this is fine.